toolkit/src/styles/components/list-item.scss

```
@@ -0,0 +1,54 @@
+/**
+* Copyright (c) HashiCorp, Inc.
+*/
+
+@import '@hashicorp/design-system-components';
+
+.cut-list-item {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
+  justify-content: space-between;
+  align-items: center;
+  padding: 12px;
+  border-radius: 6px;
+  gap: 8px;
+}
+
+button.cut-list-item.active:focus,
+a.cut-list-item.active:focus,
+button.cut-list-item.active:focus-visible,
+a.cut-list-item.active:focus-visible
```

why the duplication (and the increased specificity)?

```suggestion
.cut-list-item.active:focus,
.cut-list-item.active:focus-visible
```